keylime/tenant.py

```diff
@@ -687,6 +692,9 @@ def do_cvbulkinfo(self):
             f"/v{self.api_version}/agents/?bulk={True}&verifier={verifier_id}", timeout=self.request_timeout
         )
 
+        # Bulkinfo can be quite large. Will not jsonify it without explicity need
+        # Tenant._jsonify_response(response, True)
```

Comment:
lines 704, 707, 715, 717 all convert it to json.

For some odd reason the final return at line 734 doesn't seem to return a `response.json()` but a plain `response`. Maybe that case is never reached ? It's odd for sure if the function was to return  non-JSON there.

Reply:
This function is in a somewhat fluid state. Right now, this is one of the very few function which returns (stdout) a (properly formatted) JSON string which can be easily consumed by a tool such as `jq`. I would like to make this the standard, but will reserve it for another PR.